model/src/main/java/org/projectnessie/api/params/ReferencesParams.java

```diff
@@ -38,6 +38,7 @@
               + "- numCommitsBehind (number of commits behind the default branch)\n\n"
               + "- commitMetaOfHEAD (the commit metadata of the HEAD commit)\n\n"
               + "- commonAncestorHash (the hash of the common ancestor in relation to the default branch).\n\n"
+              + "- numTotalCommits (the total number of commits in this reference).\n\n"
               + "A returned Tag instance will only contain the 'commitMetaOfHEAD' field.\n\n"
```

Comment:
This line (and the other ones in the other classes) needs adjustment given that we're now returning `numTotalCommits` for Tags as well